### DISCUSSION TITLE
Deoptimising

### DISCUSSION THREAD

**Discussion by R0N1n-dev** [babel/babel#14788] [General] [answered]
Babel keeps deoptimising my Ui library in my vuejs project. Don't tell me it's a useless warning, coz it's totally messing up my Ui.
How do I deal with this. Ur issue template is way too verbose for this small thing. Have found no useful working solution including in ur closed issues

**Comment by nicolo-ribaudo** [comment]
Do you mean this warning?
```
Note: The code generator has deoptimised the styling of ... as it exceeds the max of 500KB
```

**Comment by R0N1n-dev** [comment]
Yes I do

**Comment by R0N1n-dev** [comment]
And it keeps only happening sometimes. Those times, my Ui becomes ugly

**Comment by nicolo-ribaudo** [comment, marked as answer]
Ok, that warning is just saying that your code will not be pretty-printed, but it will printed without worrying about the formatting. There is now way it can affect your UI, because white spaces are not significant in JavaScript (unless you are using a library that calls `.toString()` on functions and checks which whitespaces it contains: I don't think you do, and if you are doing it please stop!).

You can silence that warning using the `compact: true` generator option: if you explicitly ask Babel to print "ugly" code, it won't warn when it does so.
